howtos/weighting_scheme.rst

```diff
@@ -45,6 +85,29 @@ The UnigramLM weighting formula is based on an original approach by Bruce Croft.
 It uses statistical language modelling; 'unigram' in this case means that
 words are considered to occur independently.
 
+The Dirichlet prior method is one of the best performing language modeling approaches. Xapian
+now provides support for a modified Dirichlet prior method, namely Dir+ which is an improvement over
+the original Dirichlet prior method as it is particularly more effective across web collections
+with very long documents (where document length is much larger than average document length).
+
+TfIdfWeight
+-----------
+
+TfIdfWeight implements the support for a number of `SMART normalization variants`_ of the tf-idf
+weighting scheme. These normalizations are specified by a three character string. The first letter
+in each string specifies the normalization for the term frequency component (wdfn), the second the
+normalization for the inverse document frequency component (idfn), and the third the normalization
+used for the document weight (wtn).
+
+For more details on Xapian supported SMART tf-idf normalizations, please visit TfIdfWeight API documentation
+page `here`_. More recently supported normalization in TfIdfWeight is the pivoted (piv+) retrieval function
+which represents one of the best performing vector space models. Piv+ takes two parameters; slope and delta
+which are set to their default optimal values. You may want pass different candidate values ranging from 0.1
+to 1.5 and choose one which fits best to your system based upon corpus being used.
+
+.. _SMART normalization variants: http://nlp.stanford.edu/IR-book/html/htmledition/document-and-query-weighting-schemes-1.html
+.. _here: https://xapian.org/docs/apidoc/html/classXapian_1_1TfIdfWeight.html
```

Comment:
This isn't ideal, because the user guide can be built and used offline. I wonder if there's a suitably quick rundown of the different normalisations we can add here to give people an overview and some guidance before they break out the API docs. (Then it's less important that the link doesn't work; if offline you can still look up API docs either built or directly in the headers.)